### DISCUSSION TITLE
saving content to teensy 3.6

### DISCUSSION THREAD

**Discussion by eddybueno** [EnviralDesign/GeoPix#23] [Q&A]
noob question here im sure but I have a mobile installation that has no room for a laptop. is it possible to save content to the teensy from geopix? or would my computer always need to be connected for this setup to work? ive watched tons of tutorials and its never covered. my guess is the answer is no

**Comment by EnviralDesign** [comment]
@eddybueno great question and thanks for asking - this has been asked for quite a bit between teensy, raspberry pi and others, and is on my roadmap - but it's still going to be a ways off.

It will eventually be a workflow like this:

File -> Export -> Teensy 3.6, and this would open a teensy export window, that checks the scene and tells you what errors or issues must be corrected for it to function properly. The exported file would ideally be a hex file ready to flash to the teensy, or something easy like that. Maybe GeoPix could flash it for the user. Not sure.

At first this would be really limited, maybe a keyframed timeline show could be exported and played back. but eventually it would be interesting to have the ability to export external control over ethernet for instance, which would require a teensy to be fitted with networking hardware, etc.

The Raspberry pi would be more suited for this, but it does suffer from the issue of not being able to drive as many parallel strips as the teensy.

Anyhow, will come back to this, but not for 2-3 months at minimum.

**Reply by bostonkassidy** [reply]
This feature would be such a game changer. Any success yet writing a sequence from touch to a hex file?
